FWCore/Services/plugins/BuildFile.xml

```diff
@@ -1,6 +1,8 @@
 <use   name="FWCore/Services"/>
 <use   name="Utilities/StorageFactory"/>
+<use   name="boost"/>
 <use   name="gcc-atomic"/>
 <library   file="*.cc" name="FWCoreServicesPlugins">
   <flags   EDM_PLUGIN="1"/>
+  <flags   CXXFLAGS="-Wno-maybe-uninitialized"/>
```

Comment:
The problem is that we have `-Werror=maybe-uninitialized` so they become actual build errors.
 If you prefer I can use `-Wno-error=maybe-uninitialized` to turn them back into warnings.